app/src/main/scala/org/alephium/explorer/persistence/schema/CustomTypes.scala

```diff
@@ -130,6 +147,19 @@ trait CustomTypes extends JdbcProfile {
   implicit lazy val hashGetResult: GetResult[Hash] =
     (result: PositionedResult) => Hash.unsafe(ByteString.fromArrayUnsafe(result.nextBytes()))
 
+  implicit lazy val addressGetResult: GetResult[Address] =
+    (result: PositionedResult) => Address.unsafe(result.nextString())
```

Comment:
yes, the `as` function of slick requires a `GetResult` for each type, I'm surprise it's not automagically done since we already define the the `JdbcType` : https://github.com/alephium/explorer-backend/blob/787e60411408dab862e8c58335b7bf1dfeba2b9d/app/src/main/scala/org/alephium/explorer/persistence/schema/CustomTypes.scala#L75-L78 I already dig this a bit but haven't found yet.